src/stdune/dyn.ml

```diff
@@ -68,10 +68,9 @@ let pp_sequence start stop x ~f =
       ( Pp.concat_mapi ~sep:Pp.cut x ~f:(fun i x ->
             Pp.box
               ( ( if i = 0 then
-                  Pp.verbatim start ++ Pp.space
+                  Pp.verbatim (start ^ " ")
                 else
-                  Pp.char ';'
-                  ++ Pp.break ~nspaces:(String.length start) ~shift:0 )
+                  Pp.verbatim "; ")
```

Comment:
Should this use as many spaces as `String.length start`?

Reply:
Good call. I fixed it to use the right number of spaces.